rules/windows/process_creation/Conti_esentutl.yaml

```diff
@@ -0,0 +1,32 @@
+title: Detection of esentutl aka Extensible Storage Engine Utilities to gather credentials

```

Comment:
`title: Gather Credentials with Esentutl Aka Extensible Storage Engine Utilities`
title don't start with detect and word are capitalization